specification/cognitiveservices/data-plane/Speech/SpeakerRecognition/identification/identification-swagger.json

```diff
@@ -0,0 +1,771 @@
+                "locale": {
+                  "type": "string",
+                  "example": "en-us"
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Speaker profile created successfully. GUID is returned to reference the created profile.",
+            "headers": {
+              "location": {
+                "type": "string",
+                "description": "url location of new resource"
+              }
+            },
+            "schema": {
+              "properties": {
+                "profileId": {
+                  "type": "string",
+                  "description": "unique identifier for profile id (guid)",
+                  "example": "49a36324-fc4b-4387-aa06-090cfbf0064f"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request\n* InvalidRequest - Request body is missing.\n* InvalidRequest - Invalid JSON format.\n* InvalidRequest - Locale parameter is missing.\n* InvalidArgument - Invalid locale value. Supported locales are {comma-separated list of locales}.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "examples": {
+              "application/json": {
+                "code": "InvalidRequest",
+                "message": "Request body is missing."
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/responses/Unauthorized"
+          },
+          "415": {
+            "description": "UnsupportedMediaType  - Unsupported media type. Only 'application/JSON' is accepted.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "examples": {
+              "application/json": {
+                "code": "UnsupportedMediaType",
+                "message": "Unsupported media type. Only 'application/JSON' is accepted."
+              }
+            }
+          },
+          "429": {
+            "$ref": "#/responses/RateLimit"
+          },
+          "500": {
+            "$ref": "#/responses/ServerError"
+          }
+        }
+      },
+      "get": {
+        "description": "Retrieves a set of profiles.\nProfiles are sorted alphabetically by ProfileId",
+        "summary": "List Profile",
+        "tags": [
+          "text-independent"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "$top",
+            "description": "The number of profiles to return. Default is 100 and the maximum is 500",
+            "type": "integer"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "properties": {
+                "profiles": {
+                  "items": {
+                    "$ref": "#/definitions/ProfileInfo"
+                  }
+                },
+                "@nextLink": {
```

Comment:
@HeidiHanZhang -- please confirm that ```@nextlink``` and ```$top``` have been previously GA'd. If they have not, then please follow the guidelines for working with collections [here](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#collections)

Reply:
done